tests/e2e_tests

```diff
@@ -19,7 +19,7 @@ do
         composer install > /dev/null 2>&1
     fi
 
-    if [ -f "run_tests.bash" ]
+    if [ -f "run_test.bash" ]
```

Comment:
If you rebase on master please undo this change.

The intended filename is `run_tests.bash`, the `then` clause not running the correct file has been fixed on master.